external/fv3fit/fv3fit/pytorch/cyclegan/cyclegan_trainer.py

```diff
@@ -392,6 +491,9 @@ def train_on_batch(
                 [sample, time, tile, channel, y, x]
             real_b: a batch of data from domain B, should have shape
                 [sample, time, tile, channel, y, x]
+            training: if True, the model will be trained, otherwise we will
+                only evaluate the loss.
+            aggregator: if given, record generated results in this aggregator
```

Comment:
True, but public API is that by default nothing happens. It's an implementation detail that I make nothing happen by creating a default aggregator that never gets used, but it would be equivalent to use e.g. a NullAggregator, so it doesn't show up in the docstring.